lix/packages/fs/src/utilities/helpers.test.ts

```diff
@@ -17,11 +17,11 @@ test("not add a slash", () => {
 })
 
 test("should add leading slash", () => {
-	expect(normalizePath("project.inlang", { leadingSlash: 'always'})).toBe("/project.inlang")
+	expect(normalizePath("project.inlang", { leadingSlash: "always" })).toBe("/project.inlang")
 })
 
 test("should add trailing slash", () => {
-	expect(normalizePath("project.inlang", { trailingSlash: 'always' })).toBe("project.inlang/")
+	expect(normalizePath("project.inlang", { trailingSlash: "always" })).toBe("project.inlang/")
 })
 
 describe("keep heading slashes free for windows norm", () => {
```

Comment:
Why do we care about windows? 

Given that we control the lix environment e2e, I would spec our path implementation (as mentioned in the other comment)

Reply:
https://github.com/opral/lix-sdk/issues/30